### PR TITLE
add LambdaVersionCleanupStep

### PR DESCRIPTION
**Pull request by tylersouthwick**
* **Please check if the PR fulfills these requirements**
- [x] The commit message describes your change
- [x] Tests for the changes have been added if possible (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] Changes are mentioned in the changelog (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their setup due to this PR?)

* **Other information**:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -701,6 +701,15 @@ lambdaCleanupVerisons(
 )
 ```
 
+To discover and delete all old versions of functions created by a AWS CloudFormation stack:
+
+```groovy
+lambdaCleanupVerisons(
```
Comment (hoegertn) [inline README.md]: Typo: Versions

File: README.md [hunk 2]
```diff
@@ -686,6 +687,29 @@ String result = invokeLambda(
...
+## lambdaCleanupVersions
+
+Cleans up lambda function versions older than the daysAgo flag.
+The main use case around this is for tooling like AWS Serverless Application Model.
+It creates lambda functions, but marks them as `DeletionPolicy: Retain` so the versions are never deleted.
+Overtime, these unused versions will accumulate and the account/region might hit the limit for maximum storage of lambda functions.
+
+```groovy
+lambdaCleanupVerisons(
+	functionName: 'myLambdaFunction',
```
Comment (hoegertn) [inline README.md]: Typo: Versions

**Comment by hoegertn** [comment]
Can you also add it to the changelog, please? Sorry I did not catch this on the first review.

**Comment by tylersouthwick** [comment]
> 
> 
> Can you also add it to the changelog, please? Sorry I did not catch this on the first review.

done

**Pull request merged**